digits/inference/job.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, model, images, epoch, layers, resize=True, **kwargs):
 
     @override
     def __getstate__(self):
-        fields_to_save = ['_id', '_name']
+        fields_to_save = ['_id', '_name', 'status_history']
```

Comment:
Not for this commit. I needed for something a while back, which I no longer recall. Removing.